splunklib/__init__.py

```diff
@@ -31,5 +29,50 @@ def setup_logging(level, log_format=DEFAULT_LOG_FORMAT, date_format=DEFAULT_DATE
                         format=log_format,
                         datefmt=date_format)
 
-__version_info__ = (1, 7, 4)
+
+def ensure_binary(s, encoding='utf-8', errors='strict'):
```

Comment:
I would think a better place to put these utility methods would be the utils package .

Reply:
methods have been moved to utils